install_workflow/maui_python3_requirements.txt

```diff
@@ -1,72 +1,16 @@
-absl-py==0.6.1
-appdirs==1.4.3
-astor==0.7.1
-attrs==18.1.0
-backcall==0.1.0
+numpy==1.14.3
```

Comment:
I think it's a good idea to take the version requirement out. I don't think there is any older version specific funcitonality that we need.

Reply:
Just had to downgrade some packages as the latest ones broke the workflow

Reply:
Sweet. So that's a to-do for later then.